src/v2/providers/alerts/appDistribution.ts

```diff
@@ -71,8 +73,8 @@ export interface InAppFeedbackPayload {
   buildVersion: string;
   /** Text entered by the tester */
   text: string;
-  /** URIs to download screenshot(s) */
-  screenshotUris?: string[];
+  /** URIs to download screenshot(s). These URIs are fast expiring. */
```

Comment:
It's not enough to just change the type annotation. ESF won't send down any value in JSON if the array is empty. We're going to need to go into the actual handler code and say `event.data.payload.screenshotUris = event.data.payload.screenshotUris ?? []`

Reply:
Oh dang.  I'm not actually familiar enough with this codebase to make that change with confidence, so I thought it'd be a good idea to decouple it, so I just split out the change from optional to non-optional for this PR and put that in a separate https://github.com/firebase/firebase-functions/pull/1188